Criterion/Main/Options.hs

```diff
@@ -50,7 +50,11 @@ data MatchType = Prefix
                  -- ^ Match by prefix. For example, a prefix of
                  -- @\"foo\"@ will match @\"foobar\"@.
                | Glob
-                 -- ^ Match by Unix-style glob pattern.
+                 -- ^ Match by Unix-style glob pattern. When using this match
+                 -- type benchmark names are treated as if they were
+                 -- file-paths. For example, the glob patterns @"*/ba*"@ and
```

Comment:
In Haddock, double quotes need to be escaped with a backslash (e.g., `@\"foo\"@`), or else they'll be rendered incorrectly.

Reply:
I missed that as well :/